packages/core/src/components/DatePicker/DatePicker.stories.mdx

```diff
@@ -18,18 +18,21 @@ import { placements } from '../Popover/Popover.stories.tsx';
     }}
 />
 
-# DatePicker component
+# DatePicker Component
 
-The datepicker component allows you to select a date through input text or calendar date selection. 
+The `DatePicker` is a controlled component that allows you to select a date through input text or using a calendar dialog. 
 
-It is a controlled react component and must have the following handle state:
+Handle the state as follows:
 
 ```tsx
 const [date, setDate] = useState<Date | null>(null);
 return <DatePicker label="Select Date" value={date} onChange={setDate} />;
 ```
 
-To open the calendar, click on the calendar icon inside the DatePicker component. To close the calendar, select any date or click outside of the DatePicker component. You can also manually enter the date.
+To use the calendar below, follow the steps given:
+1. Select the calendar icon inside the DatePicker component to open the calendar.
```

Comment:
These three are not steps, they are individual statements. Also instead of using `select` word we should use `click`.